clinvar-variant-types/clinvar-variant-types.py

```diff
@@ -63,18 +63,8 @@ def review_status_stars(review_status):
 # the tree because we don't need it anymore.
 
 elements_processed = 0
-for event, elem in ElementTree.iterparse(gzip.open(args.clinvar_xml)):
-
-    # Wait until we have built a complete ClinVarSet element. Skip
-    if elem.tag != 'ClinVarSet':
-        continue
-
-    # Go to a ReferenceClinVarAssertion element. This corresponds to a single RCV record, the main unit of ClinVar.
-    # There should only be one such record per ClinVarSet.
-    rcv_records = elem.findall('ReferenceClinVarAssertion')
-    assert len(rcv_records) == 1, 'Found multiple RCV records per ClinVarSet'
-    rcv = rcv_records[0]
-    rcv_id = 'RCV{:09}'.format(int(rcv.attrib['ID']))
+for rcv in clinvar_xml_utils.iterate_rcv_from_xml(args.clinvar_xml):
+    rcv_id = 'RCV{:09}'.format(int(rcv.attrib['ID']))  # FIXME FIXME FIXME WRONG
```

Comment:
I missed this one, thanks a lot for pointing it out! The problem was using an internal ID (basically a ClinVar database primary key) instead of a proper RCV ID. Now fixed: https://github.com/EBIvariation/eva-opentargets/pull/167/commits/7b455c2a2b53219920b7c691eabe2c3d3971ff2c

Also, at this stage this script still works with XML in a more direct way without fully utilising `clinvar_xml_utils` abstractions. Created a follow up issue to address it: EBIvariation/eva-opentargets#170.